content/docs/0.8.x/operate/multi_cluster/index.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Multi-cluster setup
+description: Install Keptn control-plane and execution-plane separately.
+weight: 60
+keywords: [0.8.x-operate]
+---
+
+
+## Overview of an example setup
+
+{{< popup_image
+link="./assets/multi_cluster.png"
+caption="Multi-cluster setup"
+width="600px">}}
+
+* **Keptn Control plane**
+  * The control plane is the minimum set of components, which are required to run a Keptn and to manage projects, stages, and services, to handle events, and to provide integration points.
+  * The control plane orchestrates the task sequences defined in Shipyard, but does not actively execute the tasks.
+  * Minimum [Cluster size](./k8s_support/#control-plane)
```

Comment:
This link doesn't work for me. I think it should be `../k8s-support/#control-plane`

Reply:
fixed it.